### ISSUE TITLE
Cannot see from character view

### ISSUE THREAD

**Issue by coaltw** [LordAshes/TaleSpire-CharacterViewPlugin#2]
When I load the mod I can see the button from the radial menu but clicking the button just overlays a gray graphic on my screen and does not move my camera. I am not sure why this is happening, the mod has never worked on my computer.

**Comment by LordAshes** [comment]
Yes. One of the recent BR updates broke this plugin as well as the View
Streams plugin. I am still working on trying to fix it.

Lord Ashes

<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avg.com
<http://www.avg.com/email-signature?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>
